File: datascience/lab1/homework/q1.R
cat("1. Celsius to Fahrenheit\n")
cat("2. Fahrenheit to Celsius\n")

raw <- readline("Enter your choice (1-2, default 1): ")
choice <- if (nchar(raw) == 0) "1" else raw

if (choice == "1") {
  raw <- readline("Enter temperature in Celsius (default 25): ")
  celsius <- if (nchar(raw) == 0) 25 else as.numeric(raw)
  fahrenheit <- (celsius * 9 / 5) + 32
  cat(sprintf("%.2f C = %.2f F\n", celsius, fahrenheit))

} else if (choice == "2") {
  raw <- readline("Enter temperature in Fahrenheit (default 77): ")
  fahrenheit <- if (nchar(raw) == 0) 77 else as.numeric(raw)
  celsius <- (fahrenheit - 32) * 5 / 9
  cat(sprintf("%.2f F = %.2f C\n", fahrenheit, celsius))

} else {
  cat("Invalid choice.\n")
}

File: datascience/lab1/py/homework/q1.py
print("1. Celsius to Fahrenheit")
print("2. Fahrenheit to Celsius")

choice = input("Enter your choice (1-2, default 1): ").strip() or "1"

if choice == "1":
    celsius = float(input("Enter temperature in Celsius (default 25): ").strip() or 25)
    fahrenheit = (celsius * 9 / 5) + 32
    print(f"{celsius:.2f} C = {fahrenheit:.2f} F")
elif choice == "2":
    fahrenheit = float(input("Enter temperature in Fahrenheit (default 77): ").strip() or 77)
    celsius = (fahrenheit - 32) * 5 / 9
    print(f"{fahrenheit:.2f} F = {celsius:.2f} C")
else:
    print("Invalid choice.")
